### DISCUSSION TITLE
Catalyst 22.02rc0

### DISCUSSION THREAD

**Discussion by Scitator** [catalyst-team/catalyst#1401] [Announcements]
### Tl;dr

Beta version of Catalyst 22 edition.

- core architecture moved to [Animus](https://github.com/Scitator/animus)-like (stages were removed)
- engines moved to [Accelerate](https://github.com/huggingface/accelerate)
- config/hydra APIs deprecated in favor of [hydra-slayer](https://github.com/catalyst-team/hydra-slayer)-custom config runners
- dl-based scripts removed from the API
- self-supervised runner moved to examples - it's better to have custom still
- contrib and utils - truncated
- requirements - simplified
- codestyle moved to -l 89 (better view on 16'' screen ;) ) 

<hr /><em>This discussion was created from the release <a href='https://github.com/catalyst-team/catalyst/releases/tag/v22.02rc0'>Catalyst 22.02rc0</a>.</em>
